pydarn/plotting/rtp.py

```diff
@@ -0,0 +1,942 @@
+# Copyright (C) 2019 SuperDARN
+# Author: Marian Schmidt
+# This code is improvement based on rti.py in the DaVitpy library
+# https://github.com/vtsuperdarn/davitpy/blob/master/davitpy
+
+"""
+Range-time Parameter plots (a.k.a Intensity)
```

Comment:
'Time' should be capitalized and '(aka)' moved before 'plots'.

Reply:
done